Makefile

```diff
@@ -133,6 +135,14 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
+.PHONY: device-plugin-build
+device-plugin-build: test ## Build docker image with the manager.
+	docker build -t ${DEVICE_IMG} -f deviceplugin.Dockerfile --network="host"
```

Comment:
Not sure if @pcolledg-xilinx has suggestions about what to call "deviceplugin.Dockerfile" and where to put it?

Reply:
According the golang recommended project layout there should be a `/build/package` directory with all the packaging related stuff there.
https://github.com/golang-standards/project-layout/blob/d0cc0aa49671e3d65add3755b7a14f2df6b12374/build/README.md?plain=1#L5
How closely we follow the recommendation is a different question